### PR TITLE
Curate 257

### PR DESCRIPTION
**Pull request by msuhovec**
Updating ds3cp to latest version that gets env vras from dot_env and has large Max_keys values for GetBucketRequest()

### CODE REVIEW

File: docker/buildimage/install-go.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ GOPATH="/go"
 
 export PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
 
-mkdir -p "$GOPATH/src" "$GOPATH/bin"
+mkdir -p "$GOPATH/src" "$GOPATH/bin" "$GOPATH/scripts"
```
Comment (dbrower) [inline docker/buildimage/install-go.sh:9]: I’m curious why this is needed in the general build image we use for all the go apps. Does a build fail if it is missing?

**Review by dbrower — CHANGES_REQUESTED** [review]
Looks fine but there is a conflict so can you rebase it on master? Also had a question whether something in the build is necessary.

File: docker/buildimage/install-go.sh (hunk 1 above)
Comment (msuhovec) [reply docker/buildimage/install-go.sh]: yes

Comment (msuhovec) [reply docker/buildimage/install-go.sh]: scripts/ds3cp=/opt/bendo/scripts/ds3cp
--
75 | /bin/sh: scripts/ds3cp=/opt/bendo/scripts/ds3cp: No such file or directory
76 | make: *** [rpm] Error 127

**Pull request merged**